docs/usage/logging.md

```diff
@@ -10,6 +10,27 @@ Kubernetes uses the underlying container runtime logging, which does not persist
 * One Loki Statefulset in the `garden` namespace which contains logs for the seed cluster and one per shoot namespace which contains logs for shoot's controlplane.
 * One Grafana Deployment in `garden` namespace and two Deployments per shoot namespace (one exposed to the end users and one for the operators). Grafana is the UI component used in the logging stack.
 
+### Container Logs rotation and retention
+Container [Log rotation](https://kubernetes.io/docs/concepts/cluster-administration/logging/#log-rotation) in kubernetes describes a subtile but important implementation detail depending on the type of used high-level container runtime. When high level container runtime is used such as `docker shim`, kubernetes components do not provide any rotation or retention implementations leaving those aspects to the downstream components. When `containerd` a cri-based container runtime is used then the Kubelet provides the necessary implementation with two configuration options: `ContainerLogMaxSize` for rotation and `ContainerLogMaxFiles` for retention.
+
+#### Docker container runtime
+In this case the log rotation and retention is implemented by a logrotate service provisioned by gardener which rotates logs once 100M size is reached. Logs are compressed on daily basis and retained for a maximum period of 14 days.
+
+#### ContainerD runtime
+In this case, the Shoot may supply `ContainerLogMaxSize` and `ContainerLogMaxFiles`. Both are optional and if nothing is specified then the
+kubelet rotate on the same size `100M` as in the docker container runtime.
+```yaml
+kubernetes:
+    kubelet:
+      # accepted values are of resource.Quantity
+      containerLogMaxSize: 150Mi
+      containerLogMaxFiles: 10
+```
+
+Those values of the `ContainerLogMaxSize` and `ContainerLogMaxFiles` properties needs to be considered with care since container log files claims disk space from the host. On the oposite side ,log rotations on sizes too small may result in frequent rotations which can be missed by other components (log shippers) observing these rotations.
```

Comment:
```suggestion
Those values of the `ContainerLogMaxSize` and `ContainerLogMaxFiles` properties need to be considered with care since container log files claim disk space from the host. On the opposite side, log rotations on sizes too small may result in frequent rotations which can be missed by other components (log shippers) observing these rotations.
```